src/inc/msquic.h

```diff
@@ -612,6 +613,9 @@ typedef struct QUIC_SETTINGS {
     uint8_t RESERVED                        : 2;
     uint8_t MaxOperationsPerDrain;
     uint8_t MtuDiscoveryMissingProbeCount;
+    uint16_t MaxBindingStatelessOperations;
+    uint16_t StatelessOperationExpirationMs;
+    uint8_t CongestionControlAlgorithm;
```

Comment:
Should just be the new CC, and you also need it in the IsSet.

Reply:
This has been checkin in another PR.